packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -190,15 +194,18 @@ class CupertinoCheckbox extends StatefulWidget {
   ///
   /// If [fillColor] resolves to null for the requested state, then the fill color
   /// falls back to [activeColor] if the state includes [WidgetState.selected],
-  /// or [inactiveColor] otherwise.
+  /// or [CupertinoColors.white] with 50% opacity otherwise.
   final WidgetStateProperty<Color?>? fillColor;
 
-  /// The color used if the checkbox is inactive.
+  /// Note: [inactiveColor] is no longer being used.
```

Comment:
We try not to use "Note" in our documentation. https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-empty-prose

I think the original first sentence was fine. It's a quick preview, and they'll see it's deprecated. It's probably best to explain in the paragraph below that it currently does nothing and to point people towards using `fillColor` to control the color when unselected.